pandas/core/ops.py

```diff
@@ -452,11 +512,11 @@ def _get_op_name(op, special):
     'gt': {'op': '>',
            'desc': 'Greater than',
            'reverse': None,
-           'df_examples': None},
+           'df_examples': _gt_example_FRAME},
     'ge': {'op': '>=',
            'desc': 'Greater than or equal to',
            'reverse': None,
-           'df_examples': None}}
+           'df_examples': _ge_example_FRAME}}
```

Comment:
Any reason why you chose these two?

Reply:
This was part of the Pandas Doc Sprint some months ago. I was tasked with ge/gt functions with teammates handling others. But we can incorporate le/lt and eq/ne pairs with similar set of examples.